packages/dicom-codec/README.md

```diff
@@ -1,21 +1,28 @@
 # dicom-codec
 
-DICOM Codecs for JavaScript
-
-# Features
-
-Currently Implemented:
-- JPEG-LS decoding
-- HTJ2K encoding
-
-Short Term:
-- JPEG-LS encoding
-- HTJ2K decoding
-- JPEG2000 encoding
-- JPEG2000 decoding
-
-Longer Term:
-- the rest of the DICOM transfer syntaxes (JPEG baseline, rle, etc)
+DICOM Codecs for JavaScript (browser and node support).
+
+# Features (v0.0.10)
+| codec name        	| transferSyntaxUID(s)                                         	| decode 	| encode 	| external codec 	| js/wasm based 	|
+|-------------------	|--------------------------------------------------------------	|:------:	|:------:	|:---------------:	|:-------------:	|
+| LittleEndian      	| 1.2.840.10008.1.2 1.2.840.10008.1.2.1 1.2.840.10008.1.2.1.99 	|    -   	|    -   	|        -        	|       -       	|
+| BigEndian         	| 1.2.840.10008.1.2.2                                          	|    -   	|    -   	|        -         	|          -     	|
+| LibjpegTurbo8Bit  	| 1.2.840.10008.1.2.4.50                                       	|    X   	|    X   	|        X        	|       X       	|
```

Comment:
Using the browser decode is faster for the .50 files.  It works there, but isn't required.